nemo/collections/asr/audio_preprocessing.py

```diff
@@ -1,16 +1,17 @@
-# Copyright (C) NVIDIA CORPORATION. All Rights Reserved.
+# Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
```

Comment:
I guess it should be "2019-"

I am forgetting in my files that we got 2020 as well ;)

Reply:
will fix in next commit